db/migrate/20220319025757_create_area_vets.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAreaVets < ActiveRecord::Migration[7.0]
+  def change
+    create_table :area_vets do |t|
+      t.integer :Area_ID
+      t.string :Area_name
+      t.string :Area_funtion
+      t.string :Area_Manager
```

Comment:
As stated on the official page
<img width="670" alt="image" src="https://user-images.githubusercontent.com/68249356/159129172-c38ec3d9-a0bb-46e5-ade9-3c1ec57a8cf2.png">
and inside this [gist](https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e) the database/fields must use snakecase. In this case you're not using it since you wrote `Area_ID` instead of `area_id`. Snakecases use downcased letters

Reply:
The id is automatically created by rails.  Also there is a typo in `function`. The `area` part in the fields is unnecessary, we already know the table name which is area. Instead of function, description fits better. Manager is wrong since it must be a reference (relationship) instead of a string, containing the id of the veterinarian

Reply:
In all the tables, we're missing the releationships